tests/test_quimb_backend.py

```diff
@@ -42,7 +42,7 @@ def test_eval(nqubits: int, tolerance: float, is_mps: bool):
     init_state_tn = copy.deepcopy(init_state)
 
     # Test qibo
-    qibo.set_backend(backend=config.qibo.backend, platform=config.qibo.platform)
+    qibo.set_backend("numpy")
```

Comment:
Ok, that's even fine by me, but maybe you wanted to change
https://github.com/qiboteam/qibotn/blob/f571fc516d801664638a490c92e6e9ce24bcb93a/tests/config.py#L11

Otherwise, you can decide to drop that file, and inline the values in the tests (that is a pretty good option).

Reply:
Can drop that file as it is doing nothing. Need to modify cuquantum side also.

Reply:
Yeah, I like the latter option because it explicitly avoids the dependency on `qibojit`.